src/utils/state.mock.js

```diff
@@ -11,8 +11,8 @@ import loremIpsum from './data/lorem-ipsum.mock';
  * Example state objects for use in tests of redux-enabled components
  */
 export const mockState = {
-  lorem: getInitialState(formatData(loremIpsum)),
-  animals: getInitialState(formatData(animals))
+  lorem: getInitialState(formatData(loremIpsum), {}),
+  animals: getInitialState(formatData(animals), {})
```

Comment:
Are the second arguments needed here (since getInitialState has that defaulting to {})

Reply:
ooh good catch, I'd added these before I added the default argument. I'll remove them.